tests/spec/unit/prepare.spec.js

```diff
@@ -649,6 +649,60 @@ describe('prepare', () => {
             });
         });
 
+        it('should support Mac apps by default', () => {
+            cfg2.name = () => 'SampleApp'; // new config does *not* have a name change
+            writeFileSyncSpy.and.callThrough();
+            return updateProject(cfg2, p.locations).then(() => {
+                const xcode = require('xcode');
+                const proj = new xcode.project(p.locations.pbxproj); /* eslint new-cap : 0 */
```

Comment:
`xcode.project` shouldn't be called with `new`. Additionally, the test file imports it as

```javascript
const xcode = require('xcode');
const XcodeProject = xcode.project;
```

So you can use `XcodeProject` for consistency.

```suggestion
                const proj = XcodeProject(p.locations.pbxproj);
```

I think this will fix:

```
1) prepare updateProject method should support Mac apps by default
  Message:
    TypeError: target is not a constructor
  Stack:
    TypeError: target is not a constructor
        at <Jasmine>
        at /Users/runner/work/cordova-ios/cordova-ios/tests/spec/unit/prepare.spec.js:657:30
```